src/app/public/modules/repeater/repeater.component.ts

```diff
@@ -125,13 +145,21 @@ export class SkyRepeaterComponent implements AfterContentInit, OnChanges, OnDest
       });
     }, 0);
 
+    // Make the first item tabbable.
+    setTimeout(() => {
+      this.items.first.tabIndex = 0;
```

Comment:
What happens if the repeater component does not include any items? Will this throw an error?

Reply:
You better believe it will! Added null check. Done.